scripts/deploy_elixir.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source /scripts/setup.sh
+
+APP_NAME="${1:-}"
+SLEEP="${2:-}"
+
+if [[ -z $APP_NAME ]]; then
+  echo "Usage: $0 <env>"
+  exit 1
+fi
+
+git push -f "git@heroku.com:$APP_NAME.git" "$CIRCLE_SHA1:master"
+
+set +u
+if [[ -z $HEROKU_SKIP_ECTO_MIGRATE ]]; then
```

Comment:
I guess this is copy/pasted - but there's a bash test operator for seeing if the var is set (`-v` I think), so `[[ -v HEROKU_SKIP_ECTO_MIGRATE && -z HEROKU_SKIP_ECTO_MIGRATE ]]` means you don't need the `set +u` / `set -u` .... I think....

The other thing you can do to avoid the `set +u` thing is:
```
: ${HEROKU_SKIP_ECTO_MIGRATE:=""}
....
if [[ -z HEROKU_SKIP_ECTO_MIGRATE ]]; then
 ...
fi
```

(see the section "Parameter Expansion" in the bash(1) man page)